### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by elmobp**
Need to test the email part rest looks good

### CODE REVIEW

File: docker-compose-sy7.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+services:
+  viodata:
+    image: docker.pkg.github.com/ac3cloud/vioreport/vioreport-sy6:latest
```
Comment (arthurbarton) [inline docker-compose-sy7.yml:3]: does that need to be sy7?

File: docker-compose-sy7.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    environment:
+      location: "sy6"
```
Comment (arthurbarton) [inline docker-compose-sy7.yml:5]: here too

File: vio-data/vmware.authfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+username=svc_mon_readonly
```
Comment (arthurbarton) [inline vio-data/vmware.authfile:1]: Are these going to be SERVICES?

**Review by arthurbarton — APPROVED** [review]
Amazing stuff
I've left annoying comments inline

File: vio-data/vmware.authfile (hunk 3 above)
Comment (elmobp) [reply vio-data/vmware.authfile:1]: negative

**Pull request merged**